src/BlockTemplatesController.php

```diff
@@ -588,4 +589,20 @@ function_exists( 'wc_get_page_id' ) &&
 
 		return $is_support;
 	}
+
+	/**
+	 * Update the product archive title to "Shop".
+	 *
+	 * @param string $post_type_name Post type 'name' label.
+	 * @param string $post_type      Post type.
+	 *
+	 * @return string
+	 */
+	public function update_product_archive_title( $post_type_name, $post_type ) {
```

Comment:
Is it possible to:
* Create a more abstract method such as `update_template_title` and pass in the data needed.
* Move the method to `BlockTemplateUtils` and import into here.

Reply:
Can you clarify this comment more? I'm not sure I fully get it. This method filters the post type title to `Shop` to keep the current behavior of WooCommerce, we only need to do this for the shop page, what is the benefit of a more abstract method?

Reply:
Apologies. I can see now. This is fine.